yapf/__init__.py

```diff
@@ -30,16 +30,23 @@
 import argparse
 import logging
 import os
-import sys
+from re import match
+
+try:
+    from textwrap import dedent, indent
```

Comment:
I prefer not importing `dedent` from `textwrap`, but using `textwrap.dedent` when necessary.